package.json

```diff
@@ -109,6 +109,7 @@
     "ts-loader": "^3.2.0",
     "tslint": "5.8.0",
     "tslint-config-prettier": "1.6.0",
+    "tslint-microsoft-contrib": "^5.0.1",
```

Comment:
Could you lock down the version?

Reply:
should be good to go. I also turned on the target blank noopener rule so you should see a lot more failures from running tslint now 😃